### ISSUE TITLE
NTR: C8: octanoylcarnitine high in blood (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8558]
New term request C8: octanoylcarnitine high in blood (segal_030514174624): Acylcarnitines

**Comment by pnrobinson** [comment]
@MickeySegal

I think that Octanoyl is actually C8:0

I would introduce a separate term for increased C8 (which includes C8:0 and C8:1)

thoughts?

**Comment by MickeySegal** [comment]
I think C8 and C8:0 are synonymous.  I don't have any disease information for C8:1, but note:

C10: decanoylcarnitine
C10:1: decenoylcarnitine 
C10:2: 2-trans,4-cis-decadienoylcarnitine 

It may be wise to use C8:0 and C10:0 as the main terms and C:8 and C:10 as synonyms to make this clear.  But C:8 alone is ambiguous as to whether it include C8:1.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000272
- label: Elevated circulating octanoylcarnitine concentration
- parents: [Elevated circulating medium-chain acylcarnitine concentration (HP:0035017)](https://hpo.jax.org/app/browse/term/HP:0035017)
- pmid: 
- def: Concentration of octanoylcarnitine (C8:0) in the blood circulation above the upper limit of normal.
- comment: C8: octanoylcarnitine high in blood
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8558
- synonyms: Elevated circulating C8:0 acylcarnitine concentration(NONE)
